py/setup.py

```diff
@@ -10,7 +10,7 @@
     long_description = f.read()
 
 # Get the version from the relevant file
-with open(path.join(here, 'version.txt'), encoding='utf-8') as f:
+with open(path.join(here, 'build', 'version.txt'), encoding='utf-8') as f:
```

Comment:
`setup.py` is python code and doesn't have any information of outside world, that's why I can't use groovy `getBuildDir` method